portal/utils/updater.js

```diff
@@ -0,0 +1,29 @@
+const { exec } = require("child_process");
+
+const {createHmac, timingSafeEqual} = require("node:crypto");
+
+const logger = require("./logger")
```

Comment:
Why these redundant spaces?

Reply:
The second space was because the namespace(?) of the modules were different. first modules were global, next module is a local module. But I'll remove the first space.

Reply:
Please setup eslint and lint-staged to automatically enforce code styling errors. It takes less than 10 minutes to setup and saves the hassle of reviewing code styling during pull requests :-)